src/seedu/addressbook/data/AddressBook.java

```diff
@@ -128,4 +129,9 @@ public boolean equals(Object other) {
                         && this.allPersons.equals(((AddressBook) other).allPersons)
                         && this.allTags.equals(((AddressBook) other).allTags));
     }
-}
+
+    /**
+     * Sorts the address book based on persons' names in alphabetical order (based on ASCII value).
+     */
+    public void sort() { allPersons.sort(); }
```

Comment:
Please check the coding standards, how you write methods with a single line of code.